swaybar/bar.c

```diff
@@ -144,6 +144,22 @@ static void wl_pointer_motion(void *data, struct wl_pointer *wl_pointer,
 	bar->pointer.y = wl_fixed_to_int(surface_y);
 }
 
+static bool check_bindings(struct swaybar *bar, uint32_t x11_button,
+		uint32_t state) {
+	bool released = state == WL_POINTER_BUTTON_STATE_RELEASED;
+	for (int i = 0; i < bar->config->bindings->length; i++) {
+		struct swaybar_binding *binding = bar->config->bindings->items[i];
+		wlr_log(WLR_DEBUG, "Checking [%u, %d] against [%u, %d, %s]",
```

Comment:
Put the command in quotes. Also maybe add the word 'binding' into the message?

Though I'm not sure if I agree with having a debug message here. It might be better to only log if the binding is actually found.